NEWS.rst

```diff
@@ -27,6 +27,7 @@ New features
 Bug fixes
 ---------
 
+* #29019: Never ending loop in manager inheritance
```

Comment:
Add this to 0.18.0 instead.

Reply:
Moved.

Reply:
Managers who are inherited are now star marked in ui
![screenshot-star](https://user-images.githubusercontent.com/37613777/58172225-335b3780-7c98-11e9-89c4-62479a35ba21.png)